.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5503
```

Comment:
Don't run two live servers at the same time. Every time it changes the port.

Reply:
thanks for reminding